cpp/include/gis.hpp

```diff
@@ -54,6 +54,28 @@ gdf_column point_in_polygon( gdf_column const & polygon_latitudes,
                              gdf_column const & query_point_latitudes,
                              gdf_column const & query_point_longitudes );
 
+/**
+ * @brief Compute the perimeter of polygons on the Earth's surface using the Haversine formula.
+ * 
+ * Note: The polygons must not have holes
+ * 
+ * The polygons are defined by sets of coordinates (latitudes and longitudes),
+ * where the first and last coordinates of each polygon must have the same value (closed).
+ * 
+ * polygon_latitudes and polygon_longitudes must have equal size to 'num_polygons'.
+ *
+ * All latitudes and longitudes must have equal datatypes (for numeric operations).
+ * 
+ * @param[in] polygons_latitudes[]: set of columns with latitudes of polygons
+ * @param[in] polygons_longitudes[]: set of columns with longitudes of polygons
+ * @param[in] num_polygons: Number of polygons
+ *
+ * @returns gdf_column of perimeters with size 'num_polygons'
+ */
+gdf_column perimeter( gdf_column* polygons_latitudes[],
+					  gdf_column* polygons_longitudes[],
```

Comment:
Please change indentation to spaces, since that is what is used already in this file.

Reply:
I had a problem with some cmake buildings. So, Could I create a new PR with all these changes?.

Reply:
I don't understand the question.

Reply:
Mark we have reverted the last commit, hope this fix the issue, please lets continue the review process

Reply:
👍